autoupdater/README.md

```diff
@@ -0,0 +1,55 @@
+# AP Operator Auto Update
+
+To perform auto update, we use
+[watchtower](https://github.com/containrrr/watchtower). It
+watches `ap_operator` container and check for the docker image update.
+Once a new image is pushed to docker hub, `watchtower` pulls the new image,
+then restarts the container that it momonitors with the new docker image.
+
+Obviously this kind of update will only work when there is no config change
+required. If there is a configuration change that requires you to set a flag in the
+config, then we will need to perform a manually config change and then update.
+
+However, we try hard to make the update painless and majority of time we can let
+it auto updated.
+
+To run the auto update simply perform:
+
+
+```
+# make sure you're inside the `autoupdater` directory.
```

Comment:
uhh, where is this directory in "inside the autoupdater directory"? Is it in this repo?